pages/api/applications/csv/index.js

```diff
@@ -0,0 +1,25 @@
+import * as HttpStatus from 'http-status-codes';
```

Comment:
If we wanted, this could be `/api/csv/applications` because the endpoint is not in the public whitelist.

Reply:
Moved :)